tasks/tag-and-push.yml

```diff
@@ -21,11 +21,7 @@
   when: ecr_image_tags.0 != image_tag
 
 - name: Push image to ECR.
-  docker_image:
-    name: "{{ ecr_image_name }}:{{ image_tag }}"
-    repository: "{{ ecr_url }}/{{ ecr_image_name }}:{{ image_tag }}"
-    push: true
-    force: true
+  shell: "docker push {{ ecr_url }}/{{ ecr_image_name }}:{{ image_tag }}"
```

Comment:
Can you change this to `command`? That way ansible-lint won't complain about it :)